File: 2023/March/Day 22/seperateDigitsInArray.cpp
// Source : https://leetcode.com/problems/separate-the-digits-in-an-array/submissions/
class Solution {
public:
    vector<int> separateDigits(vector<int>& nums) {
        
        vector<int> out;
        
        for(auto&i:nums)
        {
            vector<int> temp;
            while(i)
            {
                temp.insert(temp.begin(),i%10);
                i/=10;
            }
            
            for(auto&i:temp)
            {
                out.insert(out.end(),i);
            }
        }
        
        return out;
    }
};

File: 2023/July/Day 10/seperateDigitsInArray.py
# Source : https://leetcode.com/problems/separate-the-digits-in-an-array/
class Solution:
    def separateDigits(self, nums: List[int]) -> List[int]:
        out=[];
        for element in nums:
            tmp=[]
            while(element):
                tmp.append(element%10);
                element=int(element/10)
            tmp.reverse();
            for i in tmp:
                out.append(i);
        return out;